features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Create/Components/Commons.jsx

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React, { Component } from 'react';
+import 'react-toastify/dist/ReactToastify.min.css';
+import Button from '@material-ui/core/Button';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import { withStyles } from '@material-ui/core/styles';
+import green from '@material-ui/core/colors/green';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import PropTypes from 'prop-types';
+import Alert from 'AppComponents/Shared/Alert';
+import API from 'AppData/api.js';
+import { FormattedMessage } from 'react-intl';
+import { ScopeValidation, resourceMethod, resourcePath } from 'AppData/ScopeValidation';
+import APIInputForm from 'AppComponents/Apis/Create/Endpoint/APIInputForm';
+
+
+const styles = theme => ({
+    root: {
+        flexGrow: 1,
+    },
+    paper: {
+        padding: theme.spacing.unit * 2,
+    },
+    buttonProgress: {
+        color: green[500],
+        position: 'relative',
+        marginTop: -20,
```

Comment:
Use theme.spacing.unit to define the numbers
Ex: paddingTop: theme.spacing.unit * 5
theme.spacing.unit

Reply:
Yes, I forgot to use the `theme.spacing.unit` when defining this value previously. Please replace the `-20` with `theme.spacing.unit * 5`, because the default theme spacing unit is 8px so `theme.spacing.unit * 5` value will have the same effect, similarly, we can use replace the -50 with `theme.spacing.unit * 6.25`